example/src/main/java/com/ryanharter/auto/value/gson/example/PersonTypeAdapterFactory.java

```diff
@@ -0,0 +1,18 @@
+package com.ryanharter.auto.value.gson.example;
+
+import com.google.gson.Gson;
+import com.google.gson.TypeAdapter;
+import com.google.gson.TypeAdapterFactory;
+import com.google.gson.reflect.TypeToken;
+
+public class PersonTypeAdapterFactory implements TypeAdapterFactory {
```

Comment:
Is this type needed? You can register type adapters directly without the need for a factory